code/workspaces/test/test-script/10893_STS.md

```diff
@@ -687,6 +765,20 @@ c
 | 2 | Click on the return to homepage link | If the user is not already signed in, they are taken to the homepage page <https://testlab.test-datalabs.nerc.ac.uk/>. If the user is already signed in, they are re-directed to the projects overview page |  |
 ---
 
+#### TC2.6.2.4 User can sign in  to datalabs deployed to Kubernetes
```

Comment:
I think this should be '... deployed to Cluster-as-a-Service'

Reply:
Title edited